src/framework/ui/uimodule.cpp

```diff
@@ -106,8 +108,10 @@ void UiModule::registerUiTypes()
     qmlRegisterType<QmlDialog>("MuseScore.Ui", 1, 0, "QmlDialog");
 
     qmlRegisterUncreatableType<AbstractKeyNavigation>("MuseScore.Ui", 1, 0, "AbstractKeyNavigation", "Cannot create a AbstractType");
+    qmlRegisterUncreatableType<KeyNavigationEvent>("MuseScore.Ui", 1, 0, "KeyNavigationEvent", "Cannot create a KeyNavigationEvent");
     qmlRegisterType<KeyNavigationSection>("MuseScore.Ui", 1, 0, "KeyNavigationSection");
```

Comment:
These are not UI components, they are UI infrastructure like Theme, Launcher, etc.
On the one hand, KeyNavigations themselves have no meaning, on the other hand, you can't create a "custom" KeyNavigations component and use it, for example, instead of a FlatButton, you can create a MyCoolButton and use it.